### ISSUE TITLE
Is it possible to style toolbar(header) now?

### ISSUE THREAD

**Issue by pronevich** [react-community/react-navigation#30]
Can't find any api for this purpose. Something like in react-native-navigator:

```javascript
{
  navBarTextColor: '#000000', // change the text color of the title (remembered across pushes)
  navBarBackgroundColor: '#f7f7f7', // change the background color of the nav bar (remembered across pushes)
  navBarButtonColor: '#007aff', // change the button colors of the nav bar (eg. the back button) (remembered across pushes)
  navBarHidden: false, // make the nav bar hidden
  navBarHideOnScroll: false, // make the nav bar hidden only after the user starts to scroll
  navBarTranslucent: false, // make the nav bar semi-translucent, works best with drawUnderNavBar:true
  navBarTransparent: false, // make the nav bar transparent, works best with drawUnderNavBar:true,
  topBarElevationShadowEnabled: false, // Android only, default: true. Disables TopBar elevation shadow on Lolipop and above
  navBarNoBorder: false, // hide the navigation bar bottom border (hair line). Default false
  drawUnderNavBar: false, // draw the screen content under the nav bar, works best with   navBarTranslucent:true
}

```

**Comment by grabbou** [comment]
It's been just merged for the `header` -> https://github.com/react-community/react-navigation/commit/a60607385bc1dfbd87244b543b6daaac1b073455.

`TabBar` has it already -> see `tabBarOptions.style` https://reactnavigation.org/docs/navigators/tab

**Comment by satya164** [comment]
We still need to support a `tintColor` for header to which styles the text and icon colors

**Comment by knowbody** [comment]
closing in favour of #36

**Comment by housseindjirdeh** [comment]
Forgive me if I'm doing something wrong, but since it's merged in for `header` can't we just add it to our Stack Navigation options?

```
const MyApp = StackNavigator({
  Login: {
    screen: LoginScreen,
  },
  Main: {
    screen: MainScreen,
    navigationOptions: {
      title: 'Title',
      header: {
        style: {
          backgroundColor: 'blue',
        },
      },
   },
  },
});
```

That doesn't seem to work for some reason 🤔

**Comment by satya164** [comment]
@housseindjirdeh are you using from npm or github? I don't think it's published to npm yet

**Comment by housseindjirdeh** [comment]
@satya164 ah was just thinking that might be it, thanks so much mate appreciate it

**Comment by nonameolsson** [comment]
I am also having problems to get `color` to work. Isn't this feature published to npm?

**Comment by housseindjirdeh** [comment]
@nonameolsson Yep it's not published to npm yet, but you can get it from here instead (`npm install --save https://github.com/react-community/react-navigation.git`)

**Comment by nonameolsson** [comment]
@housseindjirdeh Thanks! I the documentation at https://reactnavigation.org up-to-date? Or does it match the version published to NPM?
